src/prefect/client/orchestration.py

```diff
@@ -3898,3 +3898,25 @@ def read_task_run_states(self, task_run_id: UUID) -> List[prefect.states.State]:
         return pydantic.TypeAdapter(List[prefect.states.State]).validate_python(
             response.json()
         )
+
+    def read_deployment(
```

Comment:
Double checking this method doesn't already exist?

Reply:
on the sync client, no it doesnt